__docs__/wonder-blocks-styles/action-styles-variants.stories.tsx

```diff
@@ -0,0 +1,155 @@
+import * as React from "react";
+import type {Meta, StoryObj} from "@storybook/react";
+import info from "@phosphor-icons/core/regular/info.svg";
+import {ScenariosLayout} from "../components/scenarios-layout";
+import {addStyle, View} from "@khanacademy/wonder-blocks-core";
+import {
+    border,
+    semanticColor,
+    spacing,
+} from "@khanacademy/wonder-blocks-tokens";
+import Clickable from "@khanacademy/wonder-blocks-clickable";
+import {actionStyles} from "@khanacademy/wonder-blocks-styles";
+import IconButton from "@khanacademy/wonder-blocks-icon-button";
+
+const StyledButton = addStyle("button");
+
+/**
+ * The following stories are used to generate the pseudo states for these
+ * stories. This is only used for visual testing in Chromatic.
+ */
+export default {
+    title: "Packages / Styles / Action Styles / Action Styles - All Variants",
+    parameters: {
+        docs: {
+            autodocs: false,
+        },
+    },
+    render: () => {
+        const scenarios = [
```

Comment:
Would it helpful to have scenarios that include the disabled state (and disabled + focus state)?